toolchains/cc/README.md

```diff
@@ -0,0 +1,231 @@
+<!-- Generated with Stardoc: http://skydoc.bazel.build -->
+
+<!-- Edit this docstring and run `bazel run //docs:update-README.md` to change this repository's `README.md`. -->
```

Comment:
```suggestion
<!-- Edit the docstring in `toolchainc/cc/cc.bzl` and run `bazel run //docs:update-README.md` to change this repository's `README.md`. -->
```
Given that this comment will be copied into the generated `README.md` it may be better if it points to the actual location of the docstring to be edited.